src/test/java/lab4/game/BoardTest.java

```diff
@@ -0,0 +1,66 @@
+package lab4.game;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BoardTest {
+
+    @Test
+    void testEmptyBoardIsInProgress() {
+        Board board = new Board();
+        assertEquals(Board.Status.InProgress, board.getStatus());
```

Comment:
You should also try a board with 1 space filled and all but 1 spaces filled to verify that the status is `InProgress` for those too.